src/styles/mixins.scss

```diff
@@ -1,5 +1,14 @@
 @forward '@gravity-ui/uikit/styles/mixins.scss';
 
+@mixin cluster-full-width-area() {
+    width: unset;
+    margin-right: -40px;
+    padding-right: 40px;
+    padding-left: 20px;
+
+    transform: translateX(-20px);
+}
```

Comment:
Can you think off a better approach? It seems, there is no such problem on other pages, where paginated tables are used (Diagnostics, Node, PDisk, VDisk), so the problem is somewhere in cluster page layout

Reply:
Actually, there is a problem, but the problem is connected with `TableGroups`, your PR fixes it only for `Storage`, node for `Nodes` table. I suggest updating css for `TableGroups` themselves, instead of adding some hacks for wrappers

Reply:
I've totally reworked this PR. It seems to look much nicer now!
Stand is redeployed.